src/eos/eosEngine.ts

```diff
@@ -229,7 +229,9 @@ export class EosEngine extends CurrencyEngine<EosTools> {
     }
   }
 
-  processIncomingTransaction(action: EosTransactionSuperNode): number {
+  processIncomingTransaction(
+    action: ReturnType<typeof asHyperionTransaction>
```

Comment:
You could add these as typedefs near the cleaners themselves. That's what we normally do.